tests/qos/conftest.py

```diff
@@ -14,3 +16,80 @@ def pytest_addoption(parser):
     '''
     add_qos_sai_args(parser)
     add_dynamic_buffer_calculation_args(parser)
+
+@pytest.fixture(scope="function")
+def singleMemberPort(duthost, mg_facts):
+    '''
+        Installs static route for a port that is either a single-member lag or not part of a port channel.
+    '''
+    dst_port = None
+    # Try to find a port with no port channel
+    all_lag_members = []
+    for lag_dict in mg_facts["minigraph_portchannels"].values():
+        all_lag_members += lag_dict["members"]
```

Comment:
according to function comments, "a single-member lag" shoud be excluded, like below:

    for lag_dict in mg_facts["minigraph_portchannels"].values():
        if len(lag_dict["members"]) > 1:
            all_lag_members += lag_dict["members"]

otherwise, for topology "topo_t1-56-lag" which all ethernet is lag member, will encounter assertion error in line:

assert dst_port != None, "Failed to find an invidivual port for testing"